CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 1.3.0-alpha.4 - December 2, 2020
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
+
+### Changelog
+#### Features
+* Added an option in NavigationOptions for enable/disable route refresh.
```

Comment:
Missing PR references. Could you add the links?

Reply:
done